src/Titon/Environment/composer.json

```diff
@@ -20,6 +20,9 @@
         "titon/common": "*",
         "titon/event": "*"
     },
+    "suggest": {
+        "titon/context": "Use the IoC container to register UrlBuilder for global functions"
```

Comment:
Done. Also changed a typo, got the 2 variables backwards in the composer.json files.